reggie/install/init.sls

```diff
@@ -102,12 +102,18 @@ reggie chown {{ reggie.user }} {{ reggie[dir] }}:
       - reggie sideboard git latest
 {%- endfor %}
 
+# note: if running on windows+vagrant, virtualbox's shared win host -> linux vm FS implementation
+# doesn't support symlinks so we need to use VIRTUALENV_ALWAYS_COPY which causes file copies instead of symlinks
 reggie virtualenv:
   virtualenv.managed:
     - name: {{ reggie.install_dir }}/env
     - user: {{ reggie.user }}
     - python: /usr/bin/python3
     - system_site_packages: False
+    {% if grains['is_vagrant_windows'] | default(false) %}
```

Comment:
A better way to do this is

```
{% if grains.get('is_vagrant_windows', False) %}
```

I'll make that change in my local checkout of this branch and test it out, since I was going to up a fresh vagrant box on this branch before approving the MR anyway.